test/transformer/transformer_test.dart

```diff
@@ -75,34 +75,43 @@ main() {
     'a|web/a.txt.copy.1': 'hello',
   });
 
-  testPhases('cant overwrite files', [
-    [singleCopyTransformer]
-  ], {
-    'a|web/a.txt': 'hello',
-    'a|web/a.txt.copy': 'hello',
-  }, {}, messages: [
-    _fileExistsError("CopyBuilder", ["a|web/a.txt.copy"]),
-  ], expectBarbackErrors: true);
+  testPhases(
+      'cant overwrite files',
+      [
+        [singleCopyTransformer]
+      ],
+      {'a|web/a.txt': 'hello', 'a|web/a.txt.copy': 'hello',},
```

Comment:
If you don't mind removing the trailing commas in these map literals as well that would be cool ;)